src/measure_theory/measurable_space.lean

```diff
@@ -205,6 +205,18 @@ end
 lemma measurable_of_finite [finite α] [measurable_singleton_class α] (f : α → β) : measurable f :=
 λ s hs, (f ⁻¹' s).to_finite.measurable_set
 
+lemma measurable_of_countable [countable α] [measurable_singleton_class α] (f : α → β) :
+  measurable f :=
+begin
+  intros s hs,
+  classical,
+  have : f ⁻¹' s = ⋃ x : α, ite (f x ∈ s) ({x} : set α) ∅,
+  { ext1 x, simp, },
+  rw this,
+  refine measurable_set.Union (λ x, _),
+  exact measurable_set.ite' (λ _, measurable_set_singleton x) (λ _, measurable_set.empty),
+end
```

Comment:
Can you use [set.Union_of_singleton_coe](https://leanprover-community.github.io/mathlib_docs/data/set/lattice.html#set.Union_of_singleton_coe)? Seems we're missing an `instance [countable α] : countable ↥s`.
But if you use [set.bUnion_of_singleton](https://leanprover-community.github.io/mathlib_docs/data/set/lattice.html#set.bUnion_of_singleton) then you only need that every proposition is a subsingleton ([subsingleton_prop](https://leanprover-community.github.io/mathlib_docs/init/logic.html#subsingleton_prop)) and hence countable (should be automatic).

Reply:
I am afraid I don't follow. Where do you want me to use those lemmas?